RecoCTPPS/PixelLocal/python/CTPPSPixelClusterizer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+clusterProd = cms.EDProducer("CTPPSPixelClusterProducer",
```

Comment:
please rename this to be somewhat similar in style to other detectors or even the ctpps digis
e.g. ctppsPixelClusters
Please also rename this file to match the name of the producer